<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>1007</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>선생님용</string>
  </property>
  <widget class="QGroupBox" name="Concentration_box">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>집중도 그래프</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Graph_layout"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ClientInfo_box">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>621</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>학생별 상태</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>수업 링크</string>
   </property>
   <widget class="QTextEdit" name="code_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="message_TextBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="UserCam_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>학생 카메라</string>
   </property>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Message_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>학생별 알림</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="client_table">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>601</width>
     <height>541</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>학번</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>집중도</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>특이사항</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ip</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>학생 정보</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="student_num_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>학번:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="student_name_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>이름:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>Message_box</zorder>
  <zorder>Concentration_box</zorder>
  <zorder>ClientInfo_box</zorder>
  <zorder>groupBox</zorder>
  <zorder>message_TextBrowser</zorder>
  <zorder>UserCam_box</zorder>
  <zorder>client_table</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
